repo_policy_compliance/__init__.py

```diff
@@ -93,57 +95,112 @@ def _check_signed_commits_required(branch: Branch) -> Report:
     return Report(result=Result.PASS, reason=None)
 
 
-def all_(
-    repository_name: str,
-    source_repository_name: str,
-    target_branch_name: str,
-    source_branch_name: str,
-    commit_sha: str,
-) -> Report:
-    """Run all the checks.
+# This is a flag to indicate all policies should be checked
+class All:  # pylint: disable=too-few-public-methods
+    """Indicate that all policies should be checked."""
```

Comment:
I'm wondering if this could be an Enum, I think it would help with typing as well!

Reply:
Yeh I agree an enum would be better